### PR TITLE
add hide_labels to stacked side bar graphs

### PR DESCRIPTION
**Pull request by elrayle**

### CODE REVIEW

File: lib/gruff/bar.rb [hunk 1]
```diff
@@ -76,6 +76,14 @@ def hide_labels?
     @hide_labels
   end
 
+  def hide_left_label_area?
```
Comment (elrayle) [inline lib/gruff/bar.rb:79]: Changes in bar.rb are the same ones made in PR #450, which should make the PRs compatible for merging.

File: lib/gruff/base.rb [hunk 2]
```diff
@@ -504,6 +504,14 @@ def hide_labels?
       @hide_line_markers
     end
 
+    def hide_left_label_area?
```
Comment (elrayle) [inline lib/gruff/base.rb:507]: Changes in base.rb are the same ones made in PR #450, which should make the PRs compatible for merging.

File: test/test_sidestacked_bar.rb [hunk 3]
```diff
@@ -12,85 +12,91 @@ def setup
...
+      0 => 'September',
+      1 => 'Oct',
+      2 => 'Nov',
+      3 => 'Dec'
     }
   end
 
   def test_bar_graph
-    g = Gruff::SideStackedBar.new
+    g = setup_basic_graph
```
Comment (elrayle) [inline test/test_sidestacked_bar.rb:30]: Refactored tests to use the setup_basic_graph pattern.

**Comment by Watson1978** [comment]
Thanks!!

**Pull request merged**